content/pt-br/docs/reference/issues-security/official-cve-feed.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Feed Oficial de CVE
+weight: 25
+outputs:
+  - json
+  - html 
```

Comment:
Instead of this, hyperlink to the canonical CVE feed JSON URL (ie, `/docs/reference/issues-security/official-cve-feed/index.json`)

Reply:
```suggestion
```